concent_api/verifier/utils.py

```diff
@@ -0,0 +1,47 @@
+from base64 import b64encode
+import os
+import requests
+
+from django.conf import settings
+
+from golem_messages import message
+from golem_messages.shortcuts import dump
+
+
+def clean_directory(directory_path: str):
```

Comment:
Please add some unit tests for this function

Reply:
I was thinking about this one, but couldn't find a good way to do this. In this test almost everything would be mocked, and I would actually test `for` loop and `try/except`. Do you maybe have an idea how to test it better ?

Reply:
Hm... Not really :( I would just do sth similar to: `TestStorageClusterCertificatePathCheck` tests